ts/api/backend.ts

```diff
@@ -228,6 +266,40 @@ export function BackendClient(
     response_decoder: basicResponseDecoderWith401(SuccessResponse)
   };
 
+  const verificaRptT: VerificaRptT = {
+    method: "get",
+    url: ({ rptId }) =>
+      `/api/v1/payment-requests/${RptIdFromString.encode(rptId)}`,
+    headers: tokenHeaderProducer,
+    query: _ => ({}),
+    response_decoder: basicResponseDecoderWith401(PaymentRequestsGetResponse)
+  };
+
+  const attivaRptT: AttivaRptT = {
+    method: "post",
+    url: () => "/api/v1/payment-activations",
+    headers: composeHeaderProducers(tokenHeaderProducer, ApiHeaderJson),
+    query: () => ({}),
+    body: ({ rptId, paymentContextCode, amount }) =>
+      JSON.stringify({
+        rptId: RptIdFromString.encode(rptId),
+        codiceContestoPagamento: paymentContextCode,
+        importoSingoloVersamento: parseInt(amount, 10)
```

Comment:
But this would then expose a larger portion of code (i.e. the caller) to the existence of two different ways of representing amounts:

- `AmountInEuroCents` (from italia-ts-commons) represents the amount in cents, as a string of 2..10 characters and derives from the definitions of amounts in [this document](https://github.com/italia/pagopa-docs-formatoXML/blob/master/rst/id_versamento.rst), table 31
- `ImportoEuroCents` represents the amount as a number within the range 0, 99999999 (via `WithinRangeInteger`) and derives from the "amount" definition in some wsdl of the pagoPA "nodo" (iirc)

By having the conversion logic implemented there (not necessarily correct, it could also (more appropriately) have been `AmountInEuroCentsFromString.decode(amount) * 100`) I was trying to keep the adoption of `ImportoInEuroCents` to the smallest part possible.

This is probably symptom of a larger problem, i.e. the adoption of two different approaches to represent the same thing. Both are valid, and I believe the reason why we haven't converged to one yet is that the two types support different ranges of values. `AmountInEuroCents` can go up to 10 digits, so 99,999,999.99 EUR, while `ImportoEuroCents` can get to 999,999.99 EUR. 

I believe @LsKoder already has brought up this issue with pagoPA, but I'm not sure if this problem has already been addressed. If so, we should converge to a single representation and not have conversion problems any longer

Reply:
> But this would then expose a larger portion of code (i.e. the caller) to the existence of two different ways of representing amounts

yes indeed, my point is that this inconsistency should be (must be!) exposed, very well visible and the caller should be aware about it and should have the responsibility of handling any possible inconsistencies, conversion errors and such - burying/hiding the problem here would not certainly making it go away, instead would make people reading the calling code assuming that the API accepts a certain type when it's not